torch/_inductor/lowering.py

```diff
@@ -3402,56 +3401,6 @@ def loader(idx):
     )
 
 
-@register_lowering(aten.constant_pad_nd, type_promotion_kind=None)
-def constant_pad_nd(x, padding, fill_value=0):
-    assert (len(padding) % 2) == 0
-    if all(p == 0 for p in padding):
-        return clone(x)
-
```

Comment:
This old code code is suspiciously similar to pointwise_cat lowering.. wonder if we could unify constant_pad_nd and aten.cat implementations

Reply:
I thought making `pointwise_cat` a inductor primitive and make it a have a decomposition using `_unsafe_masked_index` just like `constant_pad_nd`, but AFAIK there's not way to call a decomposition from the lowering of `aten.cat`.